plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/cwc/commands/ActionRegistrar.kt

```diff
@@ -16,6 +16,13 @@ class ActionRegistrar {
         _messages.tryEmit(ContextMenuActionMessage(command))
     }
 
+    private val _scanMessages by lazy { MutableSharedFlow<CodeScanIssueActionMessage>(extraBufferCapacity = 10) }
```

Comment:
it is unnecessary to make a new flow. just make `_messages` of type `AmazonQMessage`